bin/run-canonical-extensions-tests.sh

```
@@ -0,0 +1,106 @@
+skipped=()
+echo -n "Looking up repositories (${#repositories[@]}): "
+for repository in ${repositories[@]}; do
+	match=$( ( gh workflow list --json path --jq '.[].path' --repo $repository | grep -E '.github/workflows/(manual-ci.yml|ci-manual.yml)' | wc -l | tr -d '[:space:]' ) || echo '0' )
+	if [[ $match == '1' ]]; then
+		filtered+=( $repository )
+	else
+		skipped+=( $repository )
+	fi
+	echo -n '.'
+done
+filtered=( $( printf '%s\n' "${filtered[@]}" | sort ) )
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)